source/jormungandr/jormungandr/default_settings.py

```diff
@@ -156,3 +156,7 @@
 USE_SERPY = boolean(os.getenv('JORMUNGANDR_USE_SERPY', False))
 
 PARSER_MAX_COUNT = int(os.getenv('JORMUNGANDR_PARSER_MAX_COUNT', 1000))
+
+if boolean(os.getenv('JORMUNGANDR_DISABLE_SQLPOOLING', False)):
```

Comment:
hmmm, i don't if it's a good idea to put a Class in a setting file, if I were you, i'm gonna keep the boolean in the setting file and use the real class in the implementation, it's up to you

Reply:
default_settings as changed a lot since it's creation, it has become more a configuration parser from envvar and less a configuration file.
When we will update the deployment of jormungandr we won't create a "settings.py" anymore and configure everything by environment variables